src/client/components/Divider.tsx

```diff
@@ -36,11 +38,50 @@ const decideSpace = (
     case 'loose':
       return displayText
         ? css`
-            margin-top: 37px;
+            margin-top: ${space[6]}px;
           `
         : css`
             margin-top: ${space[12]}px;
           `;
+    case 'none':
+      return css`
```

Comment:
Added this as an option for cases where we want to find tune the spacing between the divider and the rest of the components

Reply:
To be added to Source Kitchen instead